deploy/apecloud-mysql-cluster/values.yaml

```diff
@@ -2,66 +2,54 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
-replicaCount: 3
+name: mycluster
 
-terminationPolicy: Delete
+## @param version MySQL cluster version
+##
+version: ac-mysql-8.0.30
 
-clusterVersionOverride: ""
-nameOverride: ""
-fullnameOverride: ""
+## @param mode MySQL cluster topology mode, standalone, replication or raftGroup
+##
+mode: standalone
 
-monitor:
-  enabled: false
+## @param replicas MySQL cluster replicas, for standalone mode, replicas must be 1
+replicas: 1
 
-resources: { }
-  # We usually recommend not to specify default resources and to leave this as a conscious
-  # choice for the user. This also increases chances charts run on environments with little
-  # resources, such as Minikube. If you do want to specify resources, uncomment the following
-  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+## @param cpu
+##
+cpu: 0.5
 
-  # limits:
-  #   cpu: 500m
-  #   memory: 2Gi
-  # requests:
-  #   cpu: 100m
-  #   memory: 1Gi 
+## @param memory, the unit is Gi
+##
+memory: 1
 
+## @param storage size, the unit is Gi
+##
+storage: 20
 
-logVolume:
-  # Provide sizeLimit to emptyDir volume.
-  sizeLimit:
-  # Provide a medium for emptyDir volumes.
-  medium:
+## @param monitoring interval, 0 is no monitoring
+##
+monitoringInterval: 0
 
-persistence:
-  enabled: true
-  data:
-    storageClassName:
-    size: 1Gi
-  # separate log volume for WAL log
-  wal:
-    enabled: false
-    storageClassName:
-    size: 1Gi
-  # separate log volume for app log
-  log:
-    # If enabled, log emptyDir volume will be superseded.
-    enabled: false
-    storageClassName:
-    size: 1Gi
+## @param terminationPolicy
+terminationPolicy: Delete
+
+## @param proxyEnabled
+##
+proxyEnabled: false
```

Comment:
should user need aware of proxy settings?

Reply:
Yes, if `mode` is raftGroup, user can select to enable proxy or not. For other modes, this filed is invalid.